providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = ">= 3.0"
```

Comment:
Not `4` ?

Reply:
There are no changes that require 4.x in this module so keeping it backwards compatible for now